src/blocks/net.rs

```diff
@@ -345,30 +307,40 @@ impl NetworkDevice {
                 "Bitrate is only available for connected devices.".to_string(),
             ));
         }
-        let command = if self.wireless {
-            format!(
-                "iw dev {} link | awk '/tx bitrate/ {{print $3\" \"$4}}'",
-                self.device
-            )
-        } else {
-            format!(
-                "ethtool {} 2>/dev/null | awk '/Speed:/ {{print $2}}'",
-                self.device
-            )
-        };
-        let mut bitrate_output = Command::new("sh")
-            .args(&["-c", &command])
-            .output()
-            .block_error("net", "Failed to execute bitrate query.")?
-            .stdout;
+        if self.wireless {
+            let bitrate_output = Command::new("iw")
+                .args(&["dev", &self.device, "link"])
+                .output()
+                .block_error("net", "Failed to execute bitrate query.")?
```

Comment:
Can you please add the command name to the error like the other ones? Makes it easier to debug later.